base/src/com/google/idea/blaze/base/sync/data/BlazeDataStorage.java

```diff
@@ -25,13 +25,17 @@
 import com.intellij.openapi.application.PathManager;
 import com.intellij.openapi.project.Project;
 import java.io.File;
+import java.util.HashMap;
+import java.util.HashSet;
 import java.util.Optional;
+import java.util.Set;
```

Comment:
Please remove this import and the`import java.util.HashMap;` as they are not needed.